1.illumination_correction/loaddata_and_illum.sh

```diff
@@ -11,4 +11,6 @@ jupyter nbconvert --to script --output-dir=scripts/ *.ipynb
 # MAKE SURE TO RUN LOADDATA CSV NOTEBOOK IN JUPYTER NOTEBOOK PRIOR TO RUNNING IC DUE TO ERROR IN PYTHON SCRIPT
 
 # run Python script calculating IC functions + extract QC features with CellProfiler
-python scripts/1.cp_illum_correction.py
+python scripts/1.extract_image_quality.py
+python scripts/2.evalute_qc.py
+python scripts/3.cp_illum_correction.py
```

Comment:
Why run qc prior to IC?

Reply:
I am curious why there was an error? hmmmm

Reply:
We run QC before IC to determine the proper thresholds to identify poor-quality images that we do not want to include in the IC calculation, which can poorly impact the result.

And the error is super weird, but the post-doc from Beth's lab said it could be because this isn't packaged in PyPi yet. Not sure though!